### PR TITLE
Updates to #val_service_basics (EN)

### PR DESCRIPTION
**Pull request by MikeMcC399**
This PR updates the FAQ article https://www.coronawarn.app/en/faq/#val_service_basics "Questions and answers about certificate verification during ticketing" to address suggestions made in issue https://github.com/corona-warn-app/cwa-website/issues/2253.

The changes are intended to improve 
- text translations from German (see the original text on https://www.coronawarn.app/de/faq/#val_service_basics) and
- use of links

## Changes

1. Add "for example" to the list "In the pandemic, they therefore need reliable access management for business premises, factory halls, aircraft, trams, hotels and restaurants or events."

2. Add "Federal Ministry of Health (Bundesministerium für Gesundheit, BMG)" to explain the abbreviation "BMG"

3. Use gender general pronouns e.g. "If the buyer requires a valid certificate for the event visit, he can carry out the certificate check immediately upon purchase." instead of "he" use "they".

4. Use the standard app name which is used elsewhere in the FAQs "Corona-Warn-App".

5. Write "use encrypted communication".

6. Replace "test" by "check".

7. Link "ISO 27001" to https://en.wikipedia.org/wiki/ISO/IEC_27001.

8. Integrate the link https://ec.europa.eu/health/sites/health/files/ehealth/docs/covid-certificate_traveller-onlinebooking_en.pdf into the text.

9. Integrate the link https://github.com/eu-digital-green-certificates/dgca-validation-service into the text.

10. Link GDPR to https://en.wikipedia.org/wiki/General_Data_Protection_Regulation.

---
Internal Tracking ID: [EXPOSUREAPP-11114](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11114)

### CODE REVIEW

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
I have one change request. 
Rest of the comments are questions. 
See the in-line comments.

File: src/data/faq.json [hunk 1]
```diff
@@ -601,35 +601,33 @@
...
                             "<b>What is a DCC validation service? </b>",
-                            "Companies must comply with legal requirements to protect the health of employees and customers. In the pandemic, they therefore need reliable access management for business premises, factory halls, aircraft, trams, hotels and restaurants or events. You need to be able to verify the authenticity of Digital COVID Certificates (DCCs) quickly and easily. This is served by a Validation Service.  The BMG has decided to enable several validation services after appropriate verification.",
+                            "Companies must comply with legal requirements to protect the health of employees and customers. In the pandemic, they therefore need reliable access management for business premises, factory halls, aircraft, trams, hotels and restaurants or events for instance. You need to be able to verify the authenticity of Digital COVID Certificates (DCCs) quickly and easily. This is served by a Validation Service.  The Federal Ministry of Health (Bundesministerium für Gesundheit, BMG) has decided to enable several validation services after appropriate verification.",
```
[the remark this answers is not in the document]
Comment (MikeMcC399) [reply src/data/faq.json:604]: @Ein-Tim 
You are right. It is a bad translation of "Sie" from the German. It should have been translated to "They", not "You".
I will correct it.

File: src/data/faq.json [hunk 2]
```diff
@@ -601,35 +601,33 @@
...
+                            "The advantage can be seen, for example, in the purchase of an event ticket. If the buyer requires a valid certificate for the event visit, they can carry out the certificate check immediately upon purchase. The certificate check at the event location is omitted. This saves waiting times, avoids long queues at the entrance control and ultimately contributes to reducing the risk of infection. If the service is accessed directly from the Corona-Warn-App or, in the future, from the CovPassCheck-App, it makes it possible to validate the DCC without having to upload it to the ticket provider. Thus, the service contributes to the most data-saving use of the DCCs.",
```
[the remark this answers is not in the document]
Comment (MikeMcC399) [reply src/data/faq.json]: The English text was wrong. The German text was right and uses "Wenn der Service direkt aus der Corona Warn-App oder zukünftig aus der CovPass-App heraus aufgerufen wird, ...."

File: src/data/faq.json [hunk 3]
```diff
@@ -601,35 +601,33 @@
...
+                            "Yes. In the Corona-Warn-App and in future also in the CovPass-App, a user explicitly declares their consent before calling up the validation service. Just as with the access check, for example when entering a restaurant, the user decides for themself whether they want to prove a certificate for verification purposes or not.",
                             "<b>Is there an approval process for DCC validation service providers?</b>",
                             "Interested companies can go through a test procedure at the BMG. The procedure is open to all providers.",
                             "<b>Which companies already offer a DCC validation service?</b>",
                             "The first provider is T-Systems. The model is deliberately designed to be open to the market and allows several providers. The verification of certificates makes an important contribution to the fight against the pandemic. With competition, we want to support the fastest possible dissemination of validation services.",
```
[the remark this answers is not in the document]
Comment (MikeMcC399) [reply src/data/faq.json:630]: @Ein-Tim 
I understand that currently there is one provider and in the future there may be more if the others pass the validation. I don't see a contradiction in the text.

[the remark this answers is not in the document]
Comment (Ein-Tim) [reply src/data/faq.json:630]: Ah, okay, makes sense. Thanks.

**Review by Ein-Tim — APPROVED** [review]
Looks good from my side!

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for the PR.
@Ein-Tim Thanks for the review.

@MikeMcC399 Introducing the external links to wikipedia I would need some internal feedback, which might take some time. If you remove those for the time being, I will merge your PR and check in the meantime if we can use those. Thanks.

**Comment by MikeMcC399** [comment]
@dsarkar 
I have removed the new Wikipedia links from the PR. These are not essential, and if it means much extra coordination for you, then I would simply drop the suggestion to add them.

**Review by dsarkar — APPROVED** [review]

**Pull request merged**